app.rb

```diff
@@ -1,9 +1,15 @@
 require 'sinatra/base'
+require './lib/peep'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
+  get '/show' do
+    @peeps = Peep.all
+    erb :index
+  end
+
```

Comment:
I see you implemented the `Peep.create` method.  The controller action for that would look something like this
```
post '/peeps' do
  Peep.create(params[:peep])
  redirect '/peeps'
end
```

depending on what the form for creating peeps looks like (and how the params are structured).